test/test_helper.rb

```diff
@@ -21,3 +21,11 @@
 require 'webmock/minitest'
 
 require 'mocha/minitest'
+
+# Override the cache/config dirs with the current repo's temp folder, as we don't want tests to touch the production
```

Comment:
I have a slight concern here that this 'patch' is not in an obvious place and could cause confusion for future maintainers.

Is it needed for every test? If it's only for a handful, maybe we could make it more explicit by putting it in a separate helper and explicitly requiring it?

Reply:
That's a fair point. I would say that we would want all tests to use the TMP dir, otherwise we're risking touching production things.